### DISCUSSION TITLE
Custom names for oidc auth providers

### DISCUSSION THREAD

**Discussion by DigNZ** [pocketbase/pocketbase#3787] [Q&A]
Hi!

We use a range of identity providers like Keycloak and Authentik. They work great with the oidc support in Pocketbase but I need to hardcode the names in otherwise the listAuthMethods() just returns `oidc` . Not a big deal but it would be nice to be able to add auth methods on the server and not need to update the client to support the correct names. Is there a recommended approach for this?

**Comment by ganigeorgiev** [comment]
Hm, something like this was requested before but for a displayable vendor name.
I'll consider with v0.20/v0.21 to add a `label` prop to the `listAuthMethods` response items that could be a config option for the OIDC providers.

**Reply by ganigeorgiev** [reply]
_Just for info in case someone stumble here - new `displayName` prop was added to each `listAuthMethods()` provider item and it can be configured for the OIDC providers from the Admin UI settings. The change is currently only in the `develop` branch together with some other OAuth2 additions and will be available with the next v0.20.0 release soon._
